test/specs/workflows-spec.js

```diff
@@ -0,0 +1,94 @@
+// Copyright 2016, EMC, Inc.
+/* global describe, it, expect, before, after */
+/* eslint-disable prefer-arrow-callback */
+
+import RackHDRestAPIv1_1 from 'src-common/messengers/RackHDRestAPIv1_1';
+
+describe('Operation Center', function() {
+  this.timeout(5000);
+
+  before(function(done) {
+    const check = () => {
+      const operationsCenter = document.querySelector('.OperationsCenter'),
+            operationsCenterGraph = document.querySelector('.OperationsCenterGraph'),
+            activeOperationsList = document.querySelector('.ActiveOperationsList');
+      if (operationsCenter &&
+          operationsCenterGraph &&
+          activeOperationsList) return done();
+      setTimeout(check, 50);
+    };
+    window.location.hash = '#/oc';
+    check();
+  });
+
+  it('is ok', function() {
+    expect(true).to.be.ok;
```

Comment:
Actually this kind of unit test does in fact do something. As well as all the others. They help inform the developer that the `before` stage finished without error. Giving clear confirmation is useful.